charts/templates/server-repository/reset-job.yaml

```diff
@@ -12,19 +12,77 @@ spec:
       backoffLimit: 0
       template:
         spec:
-          serviceAccountName: {{ template "sourcify.serverRepository.serviceAccountName" . }}
+          serviceAccountName: {{ printf "%s-%s" .Chart.Name "prvw-reset-sa" | trimSuffix "-"| trunc 52 }}
           restartPolicy: Never
           containers:
           - name: reset
             image: "{{ .Values.reset.image.repository }}:{{ .Values.reset.image.tag }}"
             imagePullPolicy: {{ .Values.reset.image.pullPolicy }}
             command:
-            - /bin/sh
+            - /bin/bash
             - -c
-            - wget https://raw.githubusercontent.com/hashgraph/hedera-sourcify/main/scripts/hedera-reset.sh ; chmod +x hedera-reset.sh ; ./hedera-reset.sh previewnet
+            - POD=$(kubectl get pods -l app.kubernetes.io/name=sourcify-server-repository -n sourcify | tail -1 | awk '{print $1}'); kubectl exec -it -n sourcify $POD -- ./hedera-reset-docker.sh previewnet
+{{- end }}
+
+---
+{{- if .Values.reset.previewnet_reset.enabled }}
```

Comment:
Lots of changes for previewnet. Are they the same as testnet?
Would it be better to specify it once and then have the env be a parameter so that the logic is specified once?

Reply:
yep, the changes to this job are also reflected in the testnet setup as well